scripts/compose.py

```diff
@@ -868,6 +868,69 @@ def _content(self):
         )
         return content
 
+class OpbeansJava(Service):
+    SERVICE_PORT = 3002
+    DEFAULT_AGENT_BRANCH = "master"
+    DEFAULT_AGENT_REPO = "elastic/apm-agent-java"
+    DEFAULT_LOCAL_REPO = "."
+
+    @classmethod
+    def add_arguments(cls, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            '--opbeans-java-local-repo',
+            default=cls.DEFAULT_LOCAL_REPO,
+        )
+
+    def __init__(self, **options):
+        super().__init__(**options)
+        self.local_repo = options.get("opbeans_java_local_repo", self.DEFAULT_LOCAL_REPO)
+        self.agent_branch = options.get("opbeans_agent_branch", self.DEFAULT_AGENT_BRANCH)
+        self.agent_repo = options.get("opbeans_agent_repo", self.DEFAULT_AGENT_REPO)
+
+    def _content(self):
+        depends_on = {
+            "elasticsearch": {"condition": "service_healthy"},
+            "postgres": {"condition": "service_healthy"}
+        }
+
+        if self.options.get("with_apm-server", True):
```

Comment:
I see we do this in `OpbeansNode` too - it should be `if not self.options.get("disable_apm_server", False)`

Reply:
And that's merged now, you probably want to rebase, you'll have a failing test that point.

Reply:
@graphaelli fixed and rebased